tests/test_forum.py

```diff
@@ -244,6 +257,39 @@ async def test_unpin_all_forum_topic_messages(self, bot, forum_group_id, real_to
         )
         assert result is True, "Failed to unpin all the messages in forum topic"
 
+    async def test_edit_general_forum_topic(self, bot, forum_group_id):
+        result = await bot.edit_general_forum_topic(
+            chat_id=forum_group_id,
+            name=f"GENERAL_{datetime.datetime.now().timestamp()}",
+        )
+        assert result is True, "Failed to edit general forum topic"
+        # no way of checking the edited name, just the boolean result
+
+    @pytest.mark.xfail(reason="Can fail due to race conditions in GH actions CI")
```

Comment:
```suggestion
    @pytest.mark.xfail(reason="Can fail b/c all CI bots use the same general topic")
```

a bit more descriptive IMO :)